packets/play_clientbound.go

```diff
@@ -688,7 +690,7 @@ type SpawnPlayer struct {
 	Z        int32
 	Yaw      int8
 	Pitch    int8
-	Metadata EntityMetadata
+//	Metadata EntityMetadata
```

Comment:
Remove it?

Reply:
W ogóle to całe EntityMetadata pewnie nie działa, bo nie aktualizowane 5 lat 😄